storage_controller/src/service.rs

```diff
@@ -6795,9 +6833,15 @@ impl Service {
     fn fill_node_plan(&self, node_id: NodeId) -> Vec<TenantShardId> {
```

Comment:
I went looking at how we choose the secondary for draining. It's via `Scheduler::node_preferred`. That function still has the assumption that there's only one secondary. We should update it to compute node scores or at least choose the secondary with the home AZ.